python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -397,7 +397,7 @@ def showInfo(self, args, pipeline, graph=None):
                     continue
 
             if showCommand == "pipeline":
-                for taskDef in pipeline:
+                for taskDef in pipeline.toExpandedPipeline():
```

Comment:
I think this would be better to just be print(pipeline), as it is a good way to see what the pipeline is expanded to if there are inherits, configs, contracts, etc. What this is doing now is really just showing the tasks that are in the pipeline.

Reply:
Agree, that should look more informative, though it breaks compatibility (which I think we will survive).